include/kernels/dslash_coarse_mma.cuh

```diff
@@ -12,6 +12,14 @@
 #include <mma_tensor_op/gemm.cuh>
 #include <complex_quda.h>
 
+#if (__COMPUTE_CAPABILITY__ >= 900) && (CUDA_VERSION >= 12060)
```

Comment:
There's a bit of a design question here--in principle this type of thing should go into a target backend, but it feels like it would be an aggressive re-engineering. I still think we should pursue it, though. Thoughts @maddyscientist ?

Reply:
I guess `*_mma.cuh` is already aggressively CUDA-specific, so maybe it doesn't matter so much for now.

Reply:
This is only compiled for CUDA in the first place so yeah I don't think it matters at this point.